curiouslearning/FeedTheMonsterJS
# DRAFT: canvas to html background component (FM-295)

# Changes
- refactor background component

# How to test
- from start screen
- press anywhere in the screen to proceed to level selection
- press any levels 
- then play
- make sure background is changed accordingly

Ref: [FM-295](https://curiouslearning.atlassian.net/browse/FM-295)
